libs/shared/ui/src/lib/components/buttons/button/button.tsx

```diff
@@ -19,6 +19,7 @@ export enum ButtonStyle {
   TAB = 'tab',
   ERROR = 'error',
   CONFIRM = 'confirm',
+  DARK = 'dark',
```

Comment:
would not it be better to not add a dark style but just add CSS tailwind helper on the normal style to convert the button in dark? It's an open discussion, maybe the DARK enum is really needed.

Reply:
Or maybe it's the term DARK that is not super explicit and we should maybe choose something like: CONTRASTED or NEGATIVE or sthg else? 
wdyt?